.circleci/config.yml

```diff
@@ -279,16 +282,24 @@ jobs:
 
   android-release:
     docker:
-      - image: circleci/golang:1.13
+      - image: circleci/rust:latest
```

Comment:
Are we confident that this would not break anytime soon? Should we pin a version here?

Reply:
Yes, we use that image everywhere else as well. In this task we really don't mind using any other image, we don't require any additional language support, I just picked our most used image.